docs/helpers/Detox.md

```diff
@@ -82,7 +83,7 @@ I.appendField('name', 'davert');
 
 #### Parameters
 
--   `field` **CodeceptJS.LocatorOrString** 
+-   `field` **([string][6] \| [object][7])** 
```

Comment:
why `CodeceptJS.LocatorOrString ` was replaced to `([string][6] \| [object][7])` in this helper?

Reply:
No idea :)